sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuCoalesceBatches.scala

```diff
@@ -223,8 +223,6 @@ case class RequireSingleBatchWithFilter(filterExpression: GpuExpression)
 case class TargetSize(override val targetSizeBytes: Long)
```

Comment:
not sure if we still need `targetSizeBytes`

Reply:
I think `CoalesceSizeGoal` still needs it

Reply:
Yes, we still need targetSizeBytes.